src/profile-logic/tracks.js

```diff
@@ -236,6 +356,13 @@ export function computeLocalTracksByPid(
 ): Map<Pid, LocalTrack[]> {
   const localTracksByPid = new Map();
 
+  // find markers that might be shown
+  const timelineAwareMarkers = profile.meta.markerSchema
```

Comment:
The variables are not well named in this function in general, which doesn't make it easy to follow. In this case for example, they are not markers but marker schemas. Can you please review and fix the variable names in this function?